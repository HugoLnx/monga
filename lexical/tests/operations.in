1+2
1-2
1*4
1.+3.
1e1+3e3
0x01-0x5
0x01+0x5
