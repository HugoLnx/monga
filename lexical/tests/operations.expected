NUMBER,1
TK_PLUS,
NUMBER,2
NUMBER,1
TK_MINUS,
NUMBER,2
NUMBER,1
TK_ASTERISK,
NUMBER,4
FLOAT,1.00000
TK_PLUS,
FLOAT,3.00000
FLOAT,10.00000
TK_PLUS,
FLOAT,3000.00000
HEXADECIMAL,1
TK_MINUS,
HEXADECIMAL,5
HEXADECIMAL,1
TK_PLUS,
HEXADECIMAL,5
